aspnet/web-pages/overview/getting-started/introducing-aspnet-web-pages-2/getting-started.md

```diff
@@ -127,19 +127,19 @@ To use Web Pages, you have to be have at least Windows XP with SP3 installed, or
 
 On the [Web Pages page](../../../index.md) of the ASP.NET website, click **Install**.
 
-![ASP.NET Web site showing &quot;Install WebMatrix&quot; button](getting-started/_static/image3.png)
+![Screenshot of the A S P dot N E T website showing the Install Web Matrix for free button highlighted with a red rectangle.](getting-started/_static/image3.png)
```

Comment:
Don't change valid alt-text

Reply:
Alt text was modified to reflect Microsoft’s accessibility guidelines for alt text per https://review.docs.microsoft.com/en-us/help/contribute/contribute-accessibility-multimedia?branch=main.